x-pack/plugins/security/public/management/role_mappings/role_mappings_management_app.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { HashRouter as Router, Route, Switch, useParams } from 'react-router-dom';
+import { i18n } from '@kbn/i18n';
+import { CoreSetup } from 'src/core/public';
+import { RegisterManagementAppArgs } from '../../../../../../src/plugins/management/public';
+import { PluginStartDependencies } from '../../plugin';
+import { RolesAPIClient } from '../roles';
+import { RoleMappingsAPIClient } from './role_mappings_api_client';
+import { DocumentationLinksService } from './documentation_links';
+import { RoleMappingsGridPage } from './role_mappings_grid';
+import { EditRoleMappingPage } from './edit_role_mapping';
+
+interface CreateParams {
+  getStartServices: CoreSetup<PluginStartDependencies>['getStartServices'];
+}
+
+export const RoleMappingsManagementApp = Object.freeze({
+  id: 'role_mappings',
+  create({ getStartServices }: CreateParams) {
+    return {
+      id: this.id,
+      order: 40,
```

Comment:
__question:__ I see it was `30` in master, same as `api_keys` - was it intentional and I should return `30` back?

Reply:
Nope, not intentional! Thanks for fixing this.